brozzler/chrome.py

```diff
@@ -259,24 +254,21 @@ def _websocket_url(self, timeout_sec=60):
                 debug_info = [x for x in all_debug_info if x["url"] == "about:blank"]
 
                 if debug_info and "webSocketDebuggerUrl" in debug_info[0]:
-                    self.logger.debug("%s returned %s", json_url, raw_json)
+                    url_logger.debug("webSocketDebuggerUrl returned", raw_json=raw_json)
                     url = debug_info[0]["webSocketDebuggerUrl"]
-                    self.logger.info(
-                        "got chrome window websocket debug url %s from %s",
-                        url,
-                        json_url,
+                    url_logger.info(
+                        "got chrome window websocket debug url",
+                        debug_url=url,
                     )
                     return url
             except brozzler.ShutdownRequested:
                 raise
             except Exception as e:
                 if time.time() - self._last_warning > 30:
-                    self.logger.warning(
-                        "problem with %s (will keep trying until timeout "
-                        "of %d seconds): %s",
-                        json_url,
-                        timeout_sec,
-                        e,
+                    url_logger.warning(
+                        "problem accessing url (will keep trying until timeout)",
+                        timeout=timeout_sec,
```

Comment:
Consider including unit:
```suggestion
                        timeout_sec=timeout_sec,
```